packages/web3-errors/src/web3_error_base.ts

```diff
@@ -17,9 +17,9 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 
 /* eslint-disable max-classes-per-file */
 
-import { Web3Error as ErrorInterface } from 'web3-types';
+import { Web3Error } from 'web3-types';
 
-export abstract class Web3Error extends Error implements ErrorInterface {
+export abstract class BaseWeb3Error extends Error implements Web3Error {
```

Comment:
`BaseWeb3Error` represents base error more accurately, Thanks .but its out of scope for issue reffered in this PR, in future should we create new issue and PR for such changes. this will make PRs small and concise to review saving time,

Reply:
As agreed, I will keep it. And sorry to include changes out of scope especially since it affects lots of files.